includes/class-amp-theme-support.php

```diff
@@ -2323,14 +2263,14 @@ public static function prepare_response( $response, $args = [] ) {
 			 * non-AMP document (intentionally not valid AMP) that contains the AMP runtime and AMP components.
 			 */
 			if ( amp_is_canonical() || is_singular( AMP_Story_Post_Type::POST_TYPE_SLUG ) ) {
-				$dom->documentElement->removeAttribute( 'amp' );
-				$dom->documentElement->removeAttribute( '⚡️' );
+				$dom->documentElement->removeAttribute( Attribute::AMP );
+				$dom->documentElement->removeAttribute( Attribute::AMP_EMOJI );
```

Comment:
There is one more emoji that now needs to be accounted for, eventually. See https://github.com/ampproject/amphtml/issues/25990.